spec/browsers/pocket_casts_user_agent_spec.rb

```diff
@@ -1,81 +1,69 @@
 require 'user_agent'
 
-shared_examples 'Pocket Casts' do
+shared_examples 'Pocket Casts' do |version, platform, os, desktop|
```

Comment:
desktop argument should accept a symbol for readability.  LIke mentioned previously